server/apiRoutes.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express')
```

Comment:
Please rename this file to `api-routes.js`. We use kabab-case for all filenames.